### DISCUSSION TITLE
ComponentInternalInstance' only refers to a type, but is being used as a value here

### DISCUSSION THREAD

**Discussion by mazi233** [vuejs/rfcs#353] [General]
![image](https://user-images.githubusercontent.com/27545582/125034065-bdf29100-e0c2-11eb-9453-3ff31dba7a72.png)
![image](https://user-images.githubusercontent.com/27545582/125034088-c77bf900-e0c2-11eb-8ac7-7a6d39cfa8aa.png)

**Comment by posva** [comment]
This is not a forum for questions
